frameworks/keyed/react-zustand/src/main.jsx

```diff
@@ -211,21 +205,21 @@ const Jumbotron = memo(
 	() => true
 )
 
-const getIds = (state) => state.data.map((el) => el.id)
+const getItems = (state) => state.data
 
 const Main = () => {
 	/**
 	 * @type {string[]}
 	 */
-	const ids = useStore(getIds, shallow)
+	const items = useStore(getItems, shallow)
```

Comment:
shallow is really needed here? looks like you can omit it 

it compare each item, right?

Reply:
Hmm. Wouldn't removing it trigger a re-render any time anything in the store changes (since we're selecting a list of objects)?

Reply:
state.data === state.data will be faster

So when data changed - re-render list

When selected changed - re-render only selected

Reply:
Okay, let me quickly test that before changing this :)

Reply:
Update: removing `shallow` drops the geometric mean from 1.74 to 1.85

Reply:
Yeah, no other changes I tried gave me similar results to the one with `shallow`, so I think we should keep it. Still, not much performance difference between Hooks and Zustand. I guess the latter only shines with transient updates.